CMakeLists.txt

```diff
@@ -473,6 +473,25 @@ add_compile_definitions(FOLLY_HAVE_INT128_T=1)
 set_source(folly)
 resolve_dependency(folly)
 
+if(CMAKE_SYSTEM_NAME MATCHES "Darwin")
```

Comment:
Perhaps we should move this to before boost - since I suspect ICU_SOURCE is only set after boost build ?

Reply:
That fixed the Mac build, for Ubuntu, it seems like find package is not finding the installed ICU, so it builds a second one which has a different version and ends up trying to use a combination of both.

Reply:
My mistake, find package can find the installed ICU, we're hard coding the source to BUNDLED which forces it to build it.  Setting it to SYSTEM for ICU seems to fix it.